build_debian.sh

```diff
@@ -447,6 +447,9 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     systemd-sysv \
     ntp
 
+# Workaround to fix issue https://github.com/systemd/systemd/issues/24668.
+sudo cp files/image_config/systemd/systemd-udevd/systemd-udevd.service $FILESYSTEM_ROOT/lib/systemd/system/systemd-udevd.service
```

Comment:
You need to apply a patch to the systemd source code, and build a deb package.

Reply:
yes, this is the best way to do it. But building systemd might take long and need heavy test like you mentioned. How about we set a systemd hash that contain the fix?

Reply:
Hi @qiluo-msft , I just checked, we may use bullseye-backports. What do you think?

Reply:
Use bullseye-backports in a bullseye host is an acceptable solution.

Reply:
Any update on bullseye-backports?

Reply:
I have built an image. Now I am running regression on that image.